packages/feeds-client/package.json

```diff
@@ -46,7 +46,8 @@
     "test": "vitest",
     "test:unit": "vitest --exclude '__integration-tests__/**'",
     "test-ci": "vitest --exclude '__integration-tests__/docs-snippets/**' --coverage",
-    "test-docs-snippets": "vitest __integration-tests__/docs-snippets/**"
+    "test-docs-snippets": "vitest __integration-tests__/docs-snippets/**",
+    "postinstall": "yarn build"
```

Comment:
Package.json modified so we can install this in QA, DON'T MERGE UNTIL CHANGES IN PACKAGE.JSON ARE REVERTED